ietf/group/views.py

```diff
@@ -737,26 +737,35 @@ def dependencies(request, acronym, group_type=None):
 
     references = Q(
         Q(source__group=group) | Q(source__in=cl_docs),
-        source__type="draft",
+        source__type="draft",  # does this need to be type__in=["draft", "rfc"]? Check for informative ref rfc->draft
```

Comment:
I think likely not, but it _might_ be a change from previous behavior without it.

Since this is only used for groups, we don't have to worry about rfcs that didn't come from drafts here.

The only use such data might have to the consumer of this function is knowing that some RFC inf-reffed this group's active draft. But that's a done deal, so the group won't really be motivated to _do_ anything with the draft given this knowledge. So it's probably noise and shouldn't be on the graph in the first place.

Reply:
I looked at a handful of cases comparing against the 11.16.0 release (i.e., for posterity, the release before the one with the "new" rfc modeling) and did not see any where nodes were missing. That included some selected specifically because they had a `refinfo` relationship pointing from an RFC to a group draft.

Also, I think the only way this could miss a relationship is if there was a relationship from an RFC to a group draft without there also having been a relationship from the draft that _became_ the RFC to the group draft. Does that happen (aside from April 1 RFCs, which I feel pretty comfortable leaving off these plots except perhaps for amusement value)?